modules/tiled-gallery.php

```diff
@@ -30,3 +30,21 @@ function jetpack_tiled_gallery_configuration_load() {
 }
 
 jetpack_load_tiled_gallery();
+
+// Tile-gallery block definition can be found in wp-calypso repo
+jetpack_register_block( 'tiled-gallery', array(
+	'render_callback' => 'jetpack_tiled_gallery_load_assets'
```

Comment:
It's worth explaining why we pass a function that loads assets as a render callback - it can be confusing without reading the function body.